dashboard/src/components/AppList/AppListItem.v2.tsx

```diff
@@ -0,0 +1,59 @@
+import { bundleIcon, circleArrowIcon, ClarityIcons } from "@clr/core/icon-shapes";
+import * as React from "react";
+import helmIcon from "../../icons/helm.svg";
+import placeholder from "../../placeholder.png";
+import { IAppOverview } from "../../shared/types";
+import * as url from "../../shared/url";
+import { CdsIcon } from "../Clarity/clarity";
+import InfoCard from "../InfoCard/InfoCard.v2";
+
+import "./AppListItem.v2.css";
+
+ClarityIcons.addIcons(bundleIcon, circleArrowIcon);
+
+export interface IAppListItemProps {
+  app: IAppOverview;
+  cluster: string;
+}
+
+function AppListItem(props: IAppListItemProps) {
+  const { app, cluster } = props;
+  const icon = app.icon ? app.icon : placeholder;
+  const appStatus = app.status.toLocaleLowerCase();
+  const updateAvailable = app.updateInfo && !app.updateInfo.error && !app.updateInfo.upToDate;
+  let tag2Content;
+  if (app.updateInfo && updateAvailable) {
+    if (app.updateInfo.appLatestVersion !== app.chartMetadata.appVersion) {
+      tag2Content = `New App: ${app.updateInfo.appLatestVersion}`;
+    } else {
+      tag2Content = `New Chart: ${app.updateInfo.chartLatestVersion}`;
+    }
+  }
+  return (
+    <InfoCard
+      key={app.releaseName}
+      link={url.app.apps.get(cluster, app.namespace, app.releaseName)}
+      title={app.releaseName}
+      icon={icon}
+      info={
+        <div className="clr-row">
+          <div className={`clr-col-2 info-icon ${updateAvailable ? "is-success" : ""}`}>
+            <CdsIcon shape={updateAvailable ? "circle-arrow" : "bundle"} />
+          </div>
+          <div className="clr-col-10">
+            <span>App: {app.chartMetadata.appVersion}</span>
+            <br />
+            <span>Chart: {app.chartMetadata.version}</span>
+          </div>
+        </div>
```

Comment:
Here you can use `Row` and `Column`. In addition, I'd add an `aria-label` to mark this content as a group and notify screen readers about what they are going to read here:

```suggestion
        <div className="clr-row" aria-label="Chart version information">
          <div className={`clr-col-2 info-icon ${updateAvailable ? "is-success" : ""}`}>
            <CdsIcon shape={updateAvailable ? "circle-arrow" : "bundle"} />
          </div>
          <div className="clr-col-10">
            <span>App: {app.chartMetadata.appVersion}</span>
            <br />
            <span>Chart: {app.chartMetadata.version}</span>
          </div>
        </div>
```